controller/embedded/api-schema/entities/terminator.put.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
I had a recent discussion w/ Dariusz about supporting PUT-to-create. PUT vs. POST creates are mostly the same except PUT-to-create requires the client to supply a location (i.e. uuid) and that multiple PUTs to the same location (i.e. uuid) is idempotent. 

We don't support this now, but I am considering it if more people bring it up. The ramification is that the PUT and POST schema should be the same.

Reply:
Seems reasonable